README.md

```diff
@@ -13,7 +13,7 @@ The goal of badgeyay is to provide a simple badge generator with the following f
    
    
 To get a better idea about the working of badgeyay ,you can check out the following:
-[![Alt Watch](https://img.youtube.com/vi/Gh8j_01LIoQ/hqdefault.jpg)](https://www.youtube.com/watch?v=Gh8j_01LIoQ)
+[![Alt Watch](https://user-images.githubusercontent.com/8705386/30831526-438f8c4c-a237-11e7-83fc-c12046f12e18.png)](https://www.youtube.com/watch?v=Gh8j_01LIoQ)
```

Comment:
Please don't change anything which is not related to PR

Reply:
I think we can let this stay. As there is an issue that wants the thumbnail changed. What do you think?